rebuild-manager.js

```diff
@@ -21,6 +21,7 @@ module.exports = class RebuildManager {
     this.requests = new Requests()
     this.nextRequests = new Requests()
 
+    return
```

Comment:
TODO

Reply:
i've commented somewhere else about removing the rebuild manager

Reply:
https://github.com/ssbc/ssb-tribes/pull/96#discussion_r1365959760